sechub-shared-kernel/src/main/java/com/mercedesbenz/sechub/sharedkernel/util/ChecksumSHA256Service.java

```diff
@@ -37,13 +37,8 @@ public boolean hasCorrectChecksum(String checkSum, InputStream inputStream) {
      */
     public String createChecksum(InputStream inputStream) {
         notNull(inputStream, "inputStream may not be null");
-        MessageDigest md;
-        String algorithm = "SHA-256";
-        try {
-            md = MessageDigest.getInstance(algorithm);
-        } catch (NoSuchAlgorithmException e) {
-            throw new IllegalStateException("Algorithm not supported:" + algorithm);
-        }
+        MessageDigest md = createSHA256MessageDigest();
```

Comment:
```suggestion
        MessageDigest digest = createSHA256MessageDigest();
```

Reply:
Changed all 3 occurrences of "md" inside this class to "digest"